GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -296,6 +296,8 @@ public override HashSet<string> UpgradeBlockingProcesses
             {
                 get { return new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "GVFS.Mount", "git", "wish" }; }
             }
+
+            public override bool SupportsInlineUpgrade { get; } = true;

```

Comment:
I'm really not a fan of this notation. Can we replace the `{ get; } = true` with `=> true;` and still have it work with the `override` and `virtual` members?

Reply:
I don't feel strongly - but I updated to use the Expression body definition.